Sources/RatingSlider/Appearances.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  Appearances.swift
+//  RatingSlider
+//
+//  Created by Ignas Jasiunas on 2020-09-07.
+//  Copyright © 2020 Trafi. All rights reserved.
+//
+
+import Foundation
+
+public enum GridStyle {
+    case labeled(LabelAppearance)
+    case dotted(DotAppearance)
+
+    var hasUpperLabeledGrid: Bool {
+        guard case .dotted(let appearance) = self, appearance.labels != nil else { return false }
+        return true
+    }
+
+    var labeledGridHeight: CGFloat? {
+        guard case .dotted(let appearance) = self, let labeledAppearance = appearance.labels else { return nil }
+        return max(labeledAppearance.inactiveFont.textHeight(), labeledAppearance.activeFont.textHeight())
+    }
+}
+
+// MARK: - Label
+
+public struct LabelAppearance {
+
+    public let activeFont: UIFont
+    public let activeColor: UIColor
+    public let inactiveFont: UIFont
+    public let inactiveColor: UIColor
+
+    public init(
+        activeFont: UIFont = .systemFont(ofSize: 12, weight: .bold),
+        activeColor: UIColor = .black,
+        inactiveFont: UIFont = .systemFont(ofSize: 12, weight: .regular),
+        inactiveColor: UIColor = .lightGray
+    ) {
+        self.activeFont = activeFont
+        self.activeColor = activeColor
+        self.inactiveFont = inactiveFont
+        self.inactiveColor = inactiveColor
+    }
+
+    public static let `default`: Self = LabelAppearance()
+
+    func fontAndColor(isActive: Bool) -> (font: UIFont, color: UIColor) {
+        isActive ? (activeFont, activeColor) : (inactiveFont, inactiveColor)
+    }
+}
+
+// MARK: - Dot
+
+public struct DotAppearance {
+
+    public let activeColor: UIColor
+    public let activeSize: CGFloat
+    public let inactiveColor: UIColor
+    public let inactiveSize: CGFloat
+
+    public let labels: LabelAppearance?
+
+    public init(
+        activeColor: UIColor = .black,
+        activeSize: CGFloat = 3.0,
+        inactiveColor: UIColor = .lightGray,
+        inactiveSize: CGFloat = 3.0,
+        labels: LabelAppearance? = nil
+    ) {
+        self.activeColor = activeColor
+        self.activeSize = activeSize
+        self.inactiveColor = inactiveColor
+        self.inactiveSize = inactiveSize
+        self.labels = labels
+    }
+
+    public static let `default`: Self = DotAppearance()
+
+    func sizeAndColor(isActive: Bool) -> (size: CGFloat, color: UIColor) {
+        isActive ? (activeSize, activeColor) : (inactiveSize, inactiveColor)
+    }
+}
+
+private extension UIFont {
+
+    func textHeight(width: CGFloat = UIScreen.main.bounds.width) -> CGFloat {
```

Comment:
Doesn't seem to be used 🤔

Reply:
Here it is -> https://github.com/trafi/RatingSlider/pull/7/files#diff-127a4b836fb92be51735b942567c8005R13-R16